src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -321,13 +322,23 @@ public function register(Application $app)
         });
 
         $app['security.access_listener'] = $app->share(function ($app) {
-            return new AccessListener(
-                $app['security.token_storage'],
-                $app['security.access_manager'],
-                $app['security.access_map'],
-                $app['security.authentication_manager'],
-                $app['logger']
-            );
+            try {
+                return new AccessListener(
+                    $app['security.token_storage'],
+                    $app['security.access_manager'],
+                    $app['security.access_map'],
+                    $app['security.authentication_manager'],
+                    $app['logger']
+                );
+            } catch (Throwable $throwable) {
+                return new AccessListener(
+                    $app['security.token_storage'],
+                    $app['security.access_manager'],
+                    $app['security.access_map'],
+                    $app['security.authentication_manager'],
+                    true // Previously, the fifth argument was a logger in Symfony5 the logger was removed and replaced with an option to throw exception or not when a token is empty.
```

Comment:
shouldnt it be the opposite? It feels like symfony 5 should be the default while the old behavior to give the logger is the BC one (if exception is thrown)